test/nodes/widgets/ui_button.spec.js

```diff
@@ -118,7 +118,7 @@ describe('ui-button node', function () {
         const loadedNodes = getLoadedNodes(helper)
         loadedNodes.should.have.length(flow.length)
         loadedNodes.forEach(node => {
-            const loadedNode = flow.find(item => item.id === node.id)
+            const loadedNode = helper.getNode(node.id)
```

Comment:
@Steve-Mcl fyi - think this is the right move (as it now catches the typo I referred to in Slack), but want to be sure it's not anything silly